src/app/mandate/mandate.component.html

```diff
@@ -0,0 +1,145 @@
+<main class="b-container">
+  @if (complete) {
+    <div id="banner-thank-you"></div>
+  }
+
+<!--  @if (!complete && !noAccess && !timedOut) {-->
+<!--    <mat-spinner class="complete-spinner" color="primary"-->
+<!--                 diameter="30" aria-label="Loading donation details"></mat-spinner>-->
```

Comment:
can we remove all the stuff referencing `donation` before merging?

Reply:
@bdsl , not sure if we want to keep the spinner part when data is loading?

Reply:
I don't think we need to, if we load it in a resolver then it will be loaded before the template renders anyway.